snare/tanner_handler.py

```diff
@@ -73,18 +73,27 @@ def create_data(self, request, response_status):
         requested_name = p.sub('/', requested_name)
 
         if detection['type'] == 1:
+            possible_requests = [requested_name]
             query_start = requested_name.find('?')
             if query_start != -1:
-                requested_name = requested_name[:query_start]
-            if requested_name == '/':
-                requested_name = self.run_args.index_page
-            try:
+                possible_requests.append(requested_name[:query_start])
+
+            found = False
```

Comment:
Instead of `found` variable you can use `file_name = None`

Reply:
Yes I have updated the code.